### PR TITLE
Elasticsearch and Kibana update

### PR DESCRIPTION
**Pull request by crassirostris**
``` release-note
Updated Elasticsearch image from version 1.5.1 to version 2.4.1. Updated Kibana image from version 4.0.2 to version 4.6.1.
```

Updated es and kibana images. Made image versions match es/kibana versions they contain.

ref #19149

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34562)

### CODE REVIEW

**Comment by piosz** [comment]
@Crassirostris please fill release notes (I added template).

File: cluster/addons/fluentd-elasticsearch/kibana-image/Dockerfile [hunk 1]
```diff
@@ -15,21 +15,26 @@
 # A Dockerfile for creating a Kibana container that is designed
 # to work with Kubernetes logging.
 
-FROM java:openjdk-7-jre
-MAINTAINER Satnam Singh "satnam@google.com"
+FROM debian:jessie
```
Comment (piosz) [inline cluster/addons/fluentd-elasticsearch/kibana-image/Dockerfile]: Why do you need `debian:jessie` here?

**Review by piosz — APPROVED** [review]
LGTM, please remove binary files

File: cluster/addons/fluentd-elasticsearch/kibana-image/Dockerfile (hunk 1 above)
Comment (Crassirostris) [inline cluster/addons/fluentd-elasticsearch/kibana-image/Dockerfile]: Ack, will change to ubuntu-slim

**Comment by piosz** [comment]
Please squash commits before merge

**Pull request merged**